app/views/alaveteli_pro/batch_request_authority_searches/_public_bodies.html.erb

```diff
@@ -1,8 +1,10 @@
 <ul>
   <% public_bodies = PublicBody.with_tag(category_tag) %>
-  <% cache [:batch_builder, locale, category_tag, public_bodies] do %>
+  <% key = rails5? ? public_bodies : public_bodies.maximum(:updated_at) %>
```

Comment:
minor nitpick (1/2) - if we use `public_bodies.maximum(:updated_at)` we get a space in the cache key which appears to be valid but is a little odd e.g. `school/public_bodies/212-2019-04-23 16:02:29 +0100/5d6745201cce2d42dca055b44c33923d`

if we use `public_bodies.maximum(:updated_at).to_i` instead, we end up with something more like `school/public_bodies/212-1556031749/da061a7420597d3b51959a58ba240c97`

Reply:
Ah yes, though it would've called #to_i in the string interpolation. Changed in 4c3c23c